cmd/led_demo/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"image/color"
+	"time"
+
+	"fyne.io/fyne/v2/app"
+	"fyne.io/fyne/v2/container"
+	xwidget "fyne.io/x/fyne/widget"
+)
+
+var (
+	counter1 int
+	ledbar1  *xwidget.LedBar
+	ledbar2  *xwidget.LedBar
+)
```

Comment:
Let's not use global variables please. Use an anonymous function in the main function instead or update the run function to take in pointer parameters for the things you have to modify.